roles/create_bastion_hypershift/tasks/main.yaml

```
@@ -0,0 +1,51 @@
+---
+- name: Get ssh key of local host
+  slurp:
+    src: "~/.ssh/{{ env.ansible_key_name }}.pub"
+  register: encoded_ssh
+  delegate_to: localhost
+
+- name: Decode the ssh key
+  set_fact:
+    ssh_key: "{{encoded_ssh.content | b64decode |replace('\n','')}}"
+
+- name: Create Directory for images and bastion.ks
+  shell: mkdir -p /home/libvirt/images/
+
+- name: Create bastion.ks file
+  template:
+    src: ../create_bastion/templates/bastion-ks.cfg.j2
```

```suggestion
    src: bastion-ks.cfg.j2
```
if this template is in the templates folder of the role, all you should need is the filename

---

this template is not there in  'templates' folder of the role , i am using the template that other  existing role is using